### PR TITLE
[Core] Clean up kratos_application

### PR DESCRIPTION
**Pull request by loumalouomega**
Just that

### CODE REVIEW

File: kratos/includes/kratos_application.h [hunk 1]
```diff
@@ -211,49 +207,25 @@ class KRATOS_API(KRATOS_CORE) KratosApplication {
...
-                    *(i_variable->second));
-
-            //			KRATOS_WATCH(i_variable->second.get());
-
-            //			KRATOS_WATCH(i->second.get().Key());
-
-            //			KRATOS_WATCH(variable_key);
-
-            i->second->SetKey(variable_key);
+            it->second->SetKey(variable_key);
```
Comment (philbucher) [inline kratos/includes/kratos_application.h:220]: ping @pooyan-dadvand does this even make sense any more? I thought now the keys are compile-time constants?

**Review by philbucher — DISMISSED** [review]
approving since this is only cleanup

**Review by philbucher — DISMISSED** [review]

**Comment by philbucher** [comment]
I think it is time to compile locally ... ;)

**Comment by loumalouomega** [comment]
> I think it is time to compile locally ... ;)

:(

**Comment by loumalouomega** [comment]
@philbucher Now it works, sorry :P

**Review by philbucher — APPROVED** [review]
:)

File: kratos/includes/kratos_application.h (hunk 1 above)
Comment (pooyan-dadvand) [reply kratos/includes/kratos_application.h:220]: Not anymore. It can be removed safely.

**Pull request merged**